docs/core_modules/model_modules/llms/root.md

```diff
@@ -24,6 +24,13 @@ resp = OpenAI().complete('Paul Graham is ')
 print(resp)
 ```
 
+To save costs, you may want to use a local model.
+```python
+from llama_index import ServiceContext
+service_context = ServiceContext.from_defaults(llm="local")
```

Comment:
very nit: slightly strange that in previous example, we operate on an LLM module, but here we get a `service_context`. I think might not be clear to user what to do with it.

Reply:
moved this to the usage pattern instead